app/models/publication.rb

```diff
@@ -103,8 +103,7 @@ def self.build_new_manual_publication(pub_hash, original_source_string)
   # @return [self]
   def update_manual_pub_from_pub_hash(incoming_pub_hash, original_source_string)
     self.pub_hash = incoming_pub_hash.merge(provenance: Settings.cap_provenance)
-    match = UserSubmittedSourceRecord.find_by_source_data(original_source_string)
-    match.publication = self if match # we may still throw this out w/o saving
```

Comment:
line 107 is not actually needed and causes a problem in Rails 7.2

we are setting the relationship below in line 114: https://github.com/sul-dlss/sul_pub/pull/1741/files#diff-25593f507bc85d4e5693ced507f17f74c2ab82adacb44eb124154e1dddac7c73R114

We specify an "inverse_of" relationship which takes care of this automatically: https://github.com/sul-dlss/sul_pub/blob/main/app/models/user_submitted_source_record.rb#L5

we